BEIMA.Backend.FT/TestObjects.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Newtonsoft.Json;
+
+namespace BEIMA.Backend.FT
+{
+    public class TestObjects
+    {
+        public class Device
```

Comment:
This is for building json objects in the FTs. This class should only be used for testing purposes.